roles/fastd-config/tasks/main.yml

```diff
@@ -92,13 +92,15 @@
     group: root
     mode: 'u=rwx,g=rx,o=rx'
 
-- name: Copy fastd nodes configuration
-  template:
-    src: 'templates/fastd/fastd_nodes.conf'
-    dest: '/etc/fastd/fastd_nodes.conf'
+- name: Copy all fastd_nodes* conf files
+  copy:
+    src: "{{ item }}"
+    dest: "/etc/fastd/"
     owner: nobody
     group: root
     mode: 'u=rwx,g=rx,o=rx'
+  with_fileglob:
```

Comment:
use one template and fill it where needed with vars `{{ dinge }}`
https://docs.ansible.com/ansible-container/container_yml/template.html